modules/ossm-customizing-istio-configuration.adoc

```diff
@@ -12,7 +12,7 @@ The `values` field of the `Istio` custom resource definition, which was created
 
 . Click *Operators* -> *Installed Operators*.
 . Click *Istio* in the *Provided APIs* column.
-. Click `Istio` instance, `istio-sample` by default, in the *Name* column.
+. Click `Istio` instance, `default` by default, in the *Name* column.
```

Comment:
Reading 'default' by default feels a bit odd, though I’m not sure if it’s just me ;-) 

Just a suggestion. 
Click the Istio instance, which is named `default` by default, in the *Name* column.
Or
Click Istio instance, which is named `default`, in the *Name* column.

Feel free to ignore my comment, if you think its fine.

Reply:
No I agree. Sounded odd to me too. I will change it to: 

Click `Istio` instance, named `default`, in the Name column.